### DISCUSSION TITLE
New to Github and lost.

### DISCUSSION THREAD

**Discussion by Kmedria** [KTStephano/StratusGFX#76] [Q&A] [answered]
Hello, I am new to Github and have only coded very small projects. I successfully cloned this repository but I don't know how to configure/run cmake properly and am getting confusing errors. I was wondering if you could show me how to configure or tell me where I can learn to do so as I can't find it anywhere on Google or Youtube. Thanks in advance.

**Comment by KTStephano** [comment]
Hi yeah I'll try to help you through it. Can you tell me what your GPU is and which operating system you're on?

**Reply by Kmedria** [reply]
GPU: Nvidia Geforce GTX 1660 Ti
OS: Widows 64bit

**Reply by KTStephano** [reply]
Ok that will work. First make sure you have cmake and python 3.10

https://cmake.org/download/
(Look for Windows x64 installer)

Python 3.10 can be installed through the Windows Store

**Reply by Kmedria** [reply]
Will python 3.11.4 suffice?

**Reply by Kmedria** [reply]
The cmake 3.27 has also now finished downloading.

**Reply by KTStephano** [reply, marked as answer]
Yeah I think that one should work.

Once you have them you should be able to start the Windows Terminal, go to where you cloned StratusGFX and run

    python3 ./build_examples.py --assimp

Can you let me know what kind of errors you get with that if any?

**Reply by Kmedria** [reply]
When running cmake-gui, it kept giving an error that sdlconfig.cmake cannot be found.

**Reply by Kmedria** [reply]
The cmd is now done.

**Reply by KTStephano** [reply]
Did the cmd give any errors or did it run successfully? Check inside StratusGFX/Bin to see if there are executables in there.

I think it's possible to do this with the gui if you do the development build steps but the build_examples.py makes it a bit faster.

**Reply by Kmedria** [reply]
The cmd did not give any errors, had 1 warning and now there is a bin folder with 10 exe files.

**Reply by Kmedria** [reply]
Thank you for all your help.

**Reply by KTStephano** [reply]
Great! The only thing left would be to follow the readme step that has you download the demo data files. They are packaged separately since they are a few gigabytes.
